spec.html

```diff
@@ -23527,7 +23527,8 @@
     <!-- es6num="19.3.3" -->
     <emu-clause id="sec-properties-of-the-boolean-prototype-object">
       <h1>Properties of the Boolean Prototype Object</h1>
-      <p>The Boolean prototype object is the intrinsic object %BooleanPrototype%. The Boolean prototype object is an ordinary object. It is not a Boolean instance and does not have a [[BooleanData]] internal slot.</p>
```

Comment:
Yay semicolons! I'm all for that.

OK, I agree with the ordinary object description (it's not an exotic object). However, I don't really know exactly what it means to say that it "is a Boolean object". As far as I can tell, what makes something a Boolean object is that it has the [[BooleanValue]] internal slot, so the sentence would seem a little redundant.

However, I don't understand the concern about implementations having the right to redefine internal methods for Boolean.prototype just because it's an exotic object. Do implementations have this right on, say, Arrays?